src/crate/crash/command.py

```diff
@@ -494,9 +509,42 @@ def main():
 
     crate_hosts = [host_and_port(h) for h in args.hosts]
     error_trace = args.verbose > 0
+
+    force_passwd_prompt = args.password
+    username = args.username
+    password = None
+
+    # If no user provided check env. variables LOGNAME, USER, LNAME and USERNAME
+    if not username:
+        username = getuser()
+
+    # If password prompt is not forced try to get it from env. variable.
+    if not force_passwd_prompt:
+        password = os.environ.get('PGPASSWORD', '')
+
+    # Prompt for password immediately to avoid that the first time trying to
+    # connect to the server runs into an `Unauthorized` excpetion
+    # is_tty = False
+    if force_passwd_prompt and is_tty:
```

Comment:
this means that even if the env variable is set, we still prompt?

Reply:
good catch. no, in this case we shouldn't prompt.